Rakefile

```diff
@@ -0,0 +1,6 @@
+desc 'Generate ruby files from protocol buffers'
+task :gen do
+  Dir['proto/temporal/**/*.proto'].each do |file|
```

Comment:
gRPC tpols accepts wildcard:

```
bundle exec grpc_tools_ruby_protoc -I proto --ruby_out=lib/gen --grpc_out=lib/gen proto/temporal/api/**/*.proto
```

Reply:
I think we'd need to quote the string with the `**` to avoid the shell trying to parse it but I'll give it a go.

Reply:
This is the command I used to generate the current protos, so it does work